modules/rte_mis_gin/scss/base/_global.scss

```diff
@@ -373,6 +373,128 @@ input[type=number] {
+                                right: 0;
+                                width: 100%;
+                                height: 40%;
+                                z-index: 9999;
+                                bottom: 0;
+                                background: transparent;
+                            }
+
+                            input {
+                                display: none;
+                            }
+
+                            .class--wrapper {
+                                border: 1px solid #d4d4d8;
+                                padding: 18px;
+                                border-radius: 8px;
+                                box-shadow: 0 1px 2px rgb(20 45 82 / 2%), 0 3px 4px rgb(20 45 82 / 3%), 0 5px 8px rgb(20 45 82 / 4%);
+                                width: 100%;
+                                text-align: center;
+                            }
+
+                            .students--wrapper {
+                                text-align: center;
+                                p{
+                                    color: $black;
+                                    font-weight: 400;
+                                }
+                            }
+
+                            .form-item__label {
+                                display: flex;
+                                font-weight: 700;
+                                color: $skyblue;
+                                justify-content: center;
+                                align-items: center;
+                                flex-direction: column;
+
+                                br {
+                                    display: none;
+                                }
+                                h2 {
+                                    margin: 0;
+                                    padding-bottom: 18px;
+                                    color: $black;
+                                    font-size: 24px;
+
+                                    @media screen and (max-width: $xl) {
+                                        font-size: 20px;
+                                    }
+                                }
+                            }
+                        }
+
+                        +ul {
+                            display: flex;
+                            align-items: start;
+                            justify-content: space-between;
+                            width: 100%;
+                            list-style: none;
+                            margin: 0;
+                            column-gap: 10px;
+                            text-align: center;
+
+                            li {
+                                max-width: 10%;
+                                width: 100%;
+                                padding: 16px 0;
+                                position: relative;
+
+                                &::before{
+                                    content: 'Total Students';
```

Comment:
Can you please confirm if this will be translatable?

Reply:
Removed the code from here and added the translatable term by module file.